packages/typescript/src/components/DeclarationJSDoc.tsx

```diff
@@ -0,0 +1,18 @@
+import { Show } from "@alloy-js/core";
+import { JSDoc } from "./JSDoc.jsx";
+import { JSDocParagraph } from "./JSDocParagraph.jsx";
+
+export interface DeclarationJSDocProps {
+  doc?: string;
+}
+
+export function DeclarationJSDoc(props: DeclarationJSDocProps) {
```

Comment:
This is what the existing `JSDoc` component is supposed to be. Seems the only difference is that this takes the doc string as a prop. You can add support for that to `JSDoc` instead I think?